content/community-update/community-update/dcp-updates.md

```diff
@@ -293,12 +352,12 @@ Managed access raw data is available from the [Synapse database](https://www.syn
 
 * Raw data and contributor-generated matrices are available for 15 new projects. These single-cell data are derived from:
 * Single-cell and single-nucleus
-* 10x 3’ v2 and v3 3’ and 10x 5’ chemistry, Fluidigm C1, Smart-seq2, and sci-CAR ATAC sequencing technologies
+* 10x 3' v2 and v3 3' and 10x 5' chemistry, Fluidigm C1, Smart-seq2, and sci-CAR ATAC sequencing technologies
 * Adipose tissue, blood, bone marrow, CSF, heart, kidney, lung, oral cavity, ovary, pancreas, prostate, skeletal muscle, testis, and umbilical vein
 * Disease states including COVID-19 infection, intracranial hypertension, multiple sclerosis, and renal cell carcinoma
 * Developmental stages including fetal, child, adolescent, and adult
 * Standardized data, including aligned BAMs and cell-by-gene count matrices (Loom format), are available for 8 additional projects with data derived from:
-* 10x v2 and v3 3’ sequencing technologies
+* 10x v2 and v3 3' sequencing technologies
```

Comment:
@kayleemathews What's going on with the single quotes here? I pulled the main branch and they looked normal.

Reply:
I may have accidentally updated them with Grammarly.

Reply:
oh! That makes sense. Grammarly!! <shakes fist> Thank you!

Reply:
Are they okay like that or should I revert those changes?

Reply:
I don't see a problem leaving them. It didn't appear to affect any title searches, so I think it's ok.